src/Psy/Plugin/Manager.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Psy\Plugin;
+
+class Manager
```

Comment:
For consistency with existing managers, this should prolly be called `PluginManager`.